src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -89,6 +93,30 @@ module sonic-portchannel {
 
 		} /* end of container PORTCHANNEL */
 
+
+        container PORTCHANNEL_INTERFACE {
+
+            description "PORTCHANNEL_INTERFACE part of config_db.json";
+
+            list PORTCHANNEL_INTERFACE_IPADDR_LIST {
+
+                key "pch_name ip_prefix";
+
+                leaf pch_name {
+                    /* key elements are mandatory by default */
+                    type leafref {
+                        path "/lag:sonic-portchannel/lag:PORTCHANNEL/lag:PORTCHANNEL_LIST/lag:portchannel_name";
+                    }
+                }
+
+                leaf ip_prefix {
+                    /* key elements are mandatory by default */
+                    type inet:ip-prefix;
```

Comment:
host bits from inet:ip-prefix are cleared by libyang while loading Data Tree, which can have side effect in some cases. I think, If the user provides config in sonic_yang.json format and then we convert it to config_db.json based of the data tree.

Reply:
@praveen-li - We are not very clear on what the question/suggestion provided here ? Are you asking to change the type of ip_prefix leaf ? Can you please elaborate more on this comment ?

Reply:
https://github.com/CESNET/libyang/issues/755

Reply:
Thanks @praveen-li. Addressed your comment.